vue-frontend/src/components/home-new/UserReview.vue

```diff
@@ -1,94 +1,119 @@
 <template>
-  <div class="container overlay">
-    <div class="v2-header-3-text text-center title-width">
+  <div
```

Comment:
Live website user review section : 
![Screenshot from 2022-09-12 14-43-25](https://user-images.githubusercontent.com/94360312/189617770-5fa976ef-5c2c-4eb8-9ae5-1f95f4ce04fd.png)

and it's your IP address user review section :
![Screenshot from 2022-09-12 14-42-45](https://user-images.githubusercontent.com/94360312/189617657-a55e0a43-44f3-4f0b-b665-a8086e81a200.png)

please fix UI